samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplicationConfig.java

```diff
@@ -77,6 +77,9 @@
 
   public static final String CFG_UDF_RESOLVER = "samza.sql.udfResolver";
   public static final String CFG_FMT_UDF_RESOLVER_DOMAIN = "samza.sql.udfResolver.%s.";
+
+  public static final String CFG_WINDOW_DURATION_MS = "samza.sql.window.ms";
```

Comment:
nit: should we prefix this with groupby, ie. "samza.sql.groupby.window.ms", since this is only used by groupBy?